### PR TITLE
fix(curriculum): replace placeholder link with actual cdn link 

### LINKED ISSUE #56343: Fix broken image in demo project for recipe lab
### Describe the Issue

In the recipe page lab, when the user clicks on the `"Show Demo"` button, the demo project has a broken image link.

Screenshot 2024-09-27 at 10 50 14 PM

This is happening, because we never replaced the placeholder link with the acutal CDN link

## How to fix the problem

Here is a link to the actual lab

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/25-front-end-development/lab-recipe-page/668f08ea07b99b1f4a91acab.md

You need to updated the solution to use the correct CDN link

Here is the correct src link that should be used for that image element in the solutions section

```md
  <img src="https://cdn.freecodecamp.org/curriculum/labs/recipe.jpg" alt="a tray of chocolate chip cookies">
```

### Affected Page

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/25-front-end-development/lab-recipe-page/668f08ea07b99b1f4a91acab.md

### Your code

see explanation above

### Expected behavior

see explanation above

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by nimishgiras123**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #56343

Screenshots : 

before : 
image
after:
image

This PR addresses issue #56343 by fixing the broken image link in the demo project for the recipe lab. The placeholder image URL has been replaced with the actual CDN link to ensure that the image displays correctly.

**Changes Made:**
- Updated the image source in the HTML file.

Please review and let me know if any further changes are needed. Thank you!

### CODE REVIEW

**Review by Shashank1618 — COMMENTED** [review]
For solving this issue it is important that we have to change the given correct image link with the corrupted one and this was efficiently done by @nimishgiras123  

He is also adding a step by step instruction how you can see the changes before and after, so this makes it more easy to understand it.
 
Therefore, I hope that your PR will be merged.

**Comment by nimishgiras123** [comment]
i accidentally closed the PR. so here I'm re-opening it. please consider reviewing it again @Shashank1618

**Review by Shashank1618 — COMMENTED** [review]
Great job on efficiently addressing the issue by replacing the corrupted image link with the correct one! Special mention to @nimishgiras123 for providing clear step-by-step instructions on how to observe the changes before and after the fix. This makes it much easier for reviewers to follow and understand the modifications. Overall, this is a well-executed solution, and I believe it will contribute to resolving the problem smoothly.

I look forward to seeing this PR merged!

**Comment by a2937** [comment]
@Shashank1618 Please don't review PRs when you don't have maintainer access. That's our job.

**Comment by a2937** [comment]
> Great job on efficiently addressing the issue by replacing the corrupted image link with the correct one! Special mention to @nimishgiras123 for providing clear step-by-step instructions on how to observe the changes before and after the fix. This makes it much easier for reviewers to follow and understand the modifications. Overall, this is a well-executed solution, and I believe it will contribute to resolving the problem smoothly.
> 
> I look forward to seeing this PR merged!

We appreciate that you are trying to help our maintainer team review contributions, but without having the necessary access all this really accomplishes is creating more noise that bogs us down.

File: curriculum/challenges/english/25-front-end-development/lab-recipe-page/668f08ea07b99b1f4a91acab.md [hunk 1]
```diff
@@ -174,7 +174,7 @@ assert.isAbove($('img')[0].alt.length, 0);
   <p>Welcome to the ultimate guide for making mini chocolate chip cookies! These bite-sized treats are perfect for
     satisfying your sweet tooth without overindulging. Follow this simple recipe to create delicious,
     crispy-on-the-outside, chewy-on-the-inside mini chocolate chip cookies that everyone will love.</p>
-  <img src="cookies-image.jpg" alt="a tray of chocolate chip cookies">
+    <img src="https://cdn.freecodecamp.org/curriculum/labs/recipe.jpg" alt="Ingredients for baking: three eggs, a bowl of flour, a glass of milk, and a whisk arranged on a wooden table.">
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-recipe-page/668f08ea07b99b1f4a91acab.md:177]:
```suggestion
  <img src="https://cdn.freecodecamp.org/curriculum/labs/recipe.jpg" alt="Ingredients for baking: three eggs, a bowl of flour, a glass of milk, and a whisk arranged on a wooden table.">
```

if you can restore the indentation to what it was, there was no reason to change it, that would be better, thank you

Comment (nimishgiras123) [reply curriculum/challenges/english/25-front-end-development/lab-recipe-page/668f08ea07b99b1f4a91acab.md:177]: Thank you for the suggestion. I'll restore the indent as you said.

**Review by majestic-owl448 — APPROVED** [review]
LGTM 👍

**Review by gikf — APPROVED** [review]

**Comment by gikf** [comment]
Hi @nimishgiras123. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Comment by nimishgiras123** [comment]
@gikf thank you for the positive feedback. Can you please guide me about how can I contribute to projects for GSOC next year ?

**Comment by ilenia-magoni** [comment]
@nimishgiras123 a PR is not the right place to ask how to contribute to other projects. you should join our [Discord](https://discord.gg/KVUmVXA) server and ask there.

**Pull request merged**